quicklogic/qlf_k4n8/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+include(../common/cmake/quicklogic_qlf_arch.cmake)
+include(../common/cmake/quicklogic_qlf_device.cmake)
+include(../common/cmake/quicklogic_board.cmake)
+
+get_filename_component(FAMILY ${CMAKE_CURRENT_SOURCE_DIR} NAME)
+
+add_subdirectory(techmap)
+
+# VPR args
+set(VPR_ARCH_ARGS "\
+    --clock_modeling ideal \
```

Comment:
This seems weird?  Does this arch have no clock routing?  Please add a clear comment what is going on here.

Reply:
This architecture is based on the OpenFPGA project where for a chip the clock network is global and non-configurable hence VPR need not to worry about the routing.

Reply:
Like I said above, add a comment.

Reply:
Comment added.